tests/unit/test_configurator/test_configurator_create_init_event.py

```diff
@@ -9,35 +9,6 @@
 from .fixtures.trace import get_trace_mocks
 
 class TestConfiguratorCreateInitEvent:
-    def test_configurator_create_init_is_lambda(
-        self,
-        mocker,
-        mock_sys,
-        mock_apm_version,
-        mock_fw_versions,
-        mock_extension,
-        mock_apmconfig_enabled_is_lambda,
-    ):
-        trace_mocks = get_trace_mocks(mocker)
-
-        test_configurator = configurator.SolarWindsConfigurator()
-        result = test_configurator._create_init_event(
-            mock_extension.Reporter,
-            mock_apmconfig_enabled_is_lambda,
-        )
```

Comment:
Removed this test because function doesn't do this check anymore (see [comment](https://github.com/solarwinds/apm-python/pull/504/files#r1939864361)).